<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Export Document</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>

  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>16</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="bottomMargin">
    <number>16</number>
   </property>

   <!-- Title -->
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="text">
      <string>Export OCR Result</string>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>

   <!-- Separator -->
   <item>
    <widget class="QFrame" name="lineSeparator">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>

   <!-- Form -->
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>12</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignVCenter</set>
     </property>

     <!-- Format -->
     <item row="0" column="0">
      <widget class="QLabel" name="lblFormat">
       <property name="text">
        <string>Export format:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboFormat"/>
     </item>

     <!-- File name -->
     <item row="1" column="0">
      <widget class="QLabel" name="lblFileName">
       <property name="text">
        <string>File name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editFileName">
       <property name="placeholderText">
        <string>ocr_output</string>
       </property>
      </widget>
     </item>

     <!-- Directory -->
     <item row="2" column="0">
      <widget class="QLabel" name="lblDir">
       <property name="text">
        <string>Destination folder:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="layoutDir">
       <item>
        <widget class="QLineEdit" name="editDir"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnBrowse">
         <property name="text">
          <string>Browseâ€¦</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>

    </layout>
   </item>

   <!-- Checkbox -->
   <item>
    <widget class="QCheckBox" name="checkOpenAfterExport">
     <property name="text">
      <string>Open file after export</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>

   <!-- Buttons -->
   <item>
    <layout class="QHBoxLayout" name="layoutButtons">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="minimumWidth">
        <number>90</number>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumWidth">
        <number>90</number>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

